scio-extra/src/main/scala/com/spotify/scio/extra/csv/CsvIO.scala

```diff
@@ -88,7 +88,7 @@ import org.apache.beam.sdk.values.PCollection
  */
 object CsvIO {
 
-  private val DefaultCsvConfiguration: CsvConfiguration = CsvConfiguration(
+  val DefaultCsvConfiguration: CsvConfiguration = CsvConfiguration(
```

Comment:
shouldn't this stay private? seems like something the user shouldn't have to access

Reply:
I made it private in a previous PR but it was used in some examples where the user would `copy` to override a single value.